### ISSUE TITLE
<b:well> and <b:tabView> don't play nicely with grid system

### ISSUE THREAD

**Issue by stephanrauh** [TheCoder4eu/BootsFaces-OSP#36]
I tried to put my panelGrid in a well. The result looks a bit weird:
```XML
<b:well size="sm">
	<b:panelGrid colSpans="2,10">
		<h:outputLabel value="Value 1" />
		<b:inputText value="#{settingsBean.value1}"/>
		<h:outputLabel value="Value 2" />
		<b:inputText value="#{settingsBean.value2}"  />
		<h:outputLabel value="Checkbox 1" />
		<b:selectBooleanCheckbox value="#{settingsBean.checkbox1}" />
		<h:outputLabel value="Checkbox 2" />
		<b:selectBooleanCheckbox value="#{settingsBean.checkbox2}" />
	</b:panelGrid>
</b:well>
```

![image](https://cloud.githubusercontent.com/assets/3045767/5564539/f2766e38-8ec6-11e4-8b6f-bca4f0d68fc5.png)

Is it possible to make the grid with a well smaller?

**Comment by TheCoder4eu** [comment]
This 2 stackoverflow entries refer to old version of Bootstrap but cold lead to a solution:

http://stackoverflow.com/questions/11855086/why-do-my-twitter-bootstrap-form-fields-overflow-their-well-using-fluid-containe

http://stackoverflow.com/questions/12195931/twitter-bootstrap-stop-input-field-extending-beyond-well/12322087#12322087

**Comment by TheCoder4eu** [comment]
This is a non-blocking issue, while we think to a possible solution I postpone to v.0.7.0

**Comment by stephanrauh** [comment]
The error was on my side. At the time, I had generated a container in the panelGrid. In the meantime, I've  removed the offending tag from the PanelGridRenderer.

**Comment by TheCoder4eu** [comment]
Was it fixed already in 0.8.0 or is there something committed for 0.9.0?

**Comment by stephanrauh** [comment]
I fixed the bug on the February 27, 2015. So it has been fixed since version 0.6.5.
